### PR TITLE
Creating tribute page

### PR DESCRIPTION
**Pull request by Hasinarahman**
Description of tribute page.
 -first Creating a new repo 
-clone the repo in VS code
-creating new html file
-creating css file
-adding linters file
At the html file 
first adding structure of html 
-adding title 
-adding css file link
-in the body tag 
-adding main tag 
-in the main tag 
-adding header tag, h1 and h2 tag, p tag, image tag.
-after main tag adding footer and address tag.
than design the page in css file.
after that push all changes in github.
and creating pull request.

### CODE REVIEW

**Comment by Hasinarahman** [comment]
Hi sir,
I complete the 3 project. please check my project and give me feedback.
@ahzamir

File: tribute-page.html
Comment (ahzamir) [inline tribute-page.html:1]: - Your HTML code spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Hasinarahman 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:
- Your HTML code is not aligned correctly and clean **Check the inline comments for more details**
- Copy your HTML file code and paste it to the freecodecamp website survey form project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

SoCloseKeeganMichaelKeyGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
Screenshot 2024-05-22 221331

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Tribute Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Hasinarahman** [comment]
Hello sir **@ahzamir** ,
I aligned correctly and clean the html code and also paste in freecodecomp it dosen't have problem in it.
and at second time you can check my repo and please tell me your feedback. 
also **thanks** for helping.

**Review by ahzamir — APPROVED** [review]
### Hi @Hasinarahman 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
Screenshot 2024-05-22 221331

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Odin project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**